cookiecutter.json

```diff
@@ -0,0 +1,12 @@
+{
+    "project_name": "Boilerplate Webapp",
+    "project_slug": "{{ cookiecutter.project_name.lower()|replace(' ', '_') }}",
+    "project_module": "{{ cookiecutter.project_name.lower()|replace(' ', '_')|replace('-', '_') }}_project",
```

Comment:
use project_slug as project_module?

Reply:
Then what do we use as application_module?

Reply:
OK, cleared up in verbal discussions: ``project_module`` will default to ``project``.